base/essentials.jl

```diff
@@ -813,8 +813,7 @@ const missing = Missing()
 
 Indicate whether `x` is [`missing`](@ref).
 """
-ismissing(::Any) = false
-ismissing(::Missing) = true
+ismissing(x) = x === missing
```

Comment:
@vtjnash should I revert back to the previous `ismissing`/`isnothing` definitions ?
I believe the less method definitions the better, because we can keep to use `InterConditional` logic as far as there're ≦3 methods even after a package defines new methods for them.

xref: https://github.com/JuliaLang/julia/pull/38905/files#r552878124